src/primitive-field/primitive-field.component.ts

```diff
@@ -103,4 +103,15 @@ export class PrimitiveFieldComponent extends AbstractFieldComponent implements O
     return this.schema['disabled'] ? -1 : 1;
   }
 
+  get tooltipPosition() {
+    let tooltipPlacement = 'top';
+    let firstElem = document.getElementById('first');
+    if (firstElem) {
+      // The first top-level element will have the same bottom placement for all it's elements.
+      if (firstElem.querySelector(`[id='${this.pathString}']`)) {
```

Comment:
what we could do is put the `pathString` of first top-level property to `app-globals` then here we can check:
```typescript
if (this.pathString.startsWith(this.appGlobalsService.firstFieldPathString)
```

Naming could be changed, but this is gonna remove all queries.

@zzacharo  @jmartinm